app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :comments_counter
```

Comment:
- [ ]  You can use the after_save to call the #update_comments_counter method every time a comment has been added 👍🏿